content/refguide/call-web-service-action.md

```diff
@@ -62,14 +62,13 @@ When consuming a WSDL that uses encoding, turning on validation will lead to a c
 
 {{% /alert %}} 
 
-When this setting is disabled, schema validation is not possible. This happens in two cases:
+Since Mendix 7.20, the schema validation for calling web service using the `new (optimized) implementation` is supported.
 
-* When you have configured the [consumed web service](consumed-web-service) to send binary data as attachments
-* When you have configured web service calls to use the new (optimized) implementation in the [project's runtime settings](project-settings)
+The schema validation is not supported when have configured the [consumed web service](consumed-web-service) to send binary data as attachments.
 
 {{% alert type="info" %}}
 
-The new (optimized) implementation feature was introduced in version 7.15.0.
+The `new (optimized) implementation` feature was introduced in version 7.15.0.

```

Comment:
Why add the backticks? Those are meant for code, right?

Reply:
Thosebacktick will show it in a box. I like to show my settings in a box. Other team does that too e.g.
Before switching to the Database option, the `DataStorage.EnableDiagnostics` custom runtime setting can be used to generate a uniqueness violation report. The unique constraint migration will need to be done if the generated report shows violations.

On the second thought, I will change it into a link instead since I removed the link.

Reply:
This acts as a link now.